src/app/progress/Progress.tsx

```diff
@@ -266,6 +278,9 @@ class Progress extends React.Component<Props, State> {
               <PageWithMenu.MenuButton href="#milestones" onClick={goToSection}>
                 <span>{t('Progress.Milestones')}</span>
               </PageWithMenu.MenuButton>
+              <PageWithMenu.MenuButton href="#Raids" onClick={goToSection}>
```

Comment:
<PageWithMenu.MenuButton href="#raids" onClick={goToSection}>

Reply:
thank you!